api/src/main/java/io/strimzi/kafka/access/model/KafkaAccessStatus.java

```diff
@@ -20,7 +19,7 @@
     editableEnabled = false,
     builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
-public class KafkaAccessStatus extends ObservedGenerationAwareStatus {
```

Comment:
I didn't review the whole upgrade guide, but the `ObservedGenerationAwareStatus` is not available in v5?
I think it would be great to have replacement, as the `observedGeneration` is something that should be kept in the status.

Reply:
I see they are mentioning to look on the following integration tests and the classes - the one which is IMHO for us is this one - https://github.com/operator-framework/java-operator-sdk/blob/1635c9ea338f8e89bacc547808d2b409de8734cf/operator-framework/src/test/java/io/javaoperatorsdk/operator/baseapi/manualobservedgeneration/ManualObservedGenerationCustomResource.java.

Ah, I guess you are missing the getter there for the observed generation. I guess that is also needed, because you should have the `observedGeneration` inside the CRD IMHO as well.